daffodil-core/src/main/scala/org/apache/daffodil/core/grammar/ElementBaseGrammarMixin.scala

```diff
@@ -700,6 +701,30 @@ trait ElementBaseGrammarMixin
       ConvertZonedCombinator(this, stringValue, textZonedConverter)
     }
 
+  /**
+   * True if the encoding is known to be an EBCDIC one, as in the encoding is not  a runtime expression
+   * and it's some ebcdic flavor. If it's any ascii flavor or a runtime expression this is false.
+   */
+  lazy val isKnownEBCDICEncoding: Boolean = {
+    charsetEv.optConstant
+      .map { bcs: BitsCharset =>
+        val nom = bcs.name.toUpperCase()
+        val res = (nom == "IBM037") ||
+          nom.startsWith("EBCDIC")
```

Comment:
This may not be sufficient to recognize EBCDIC. There are numrous IBM code pages that are EBCDIC besides IBM037. There are ones for the international variants of EBCDIC. All of them do the exact same thing for zoned numbers. 

We need something that will work regardless of the aliases vs. the "official" name. If the name, or any alias contains "ebcdic" (case insensitive) then it is an ebcdic variant.

Reply:
Should we add an `isEBCDIC` member to `BitsCharset` that defaults to `false`, and any EBCDIC encodings we implement would override that to `true`? Then this just becomes

```scala
charsetEv.optConstant.map { _.isEBCDIC }.getOrElse(false)
```

Also, doing some grepping for EBCDIC I see we also have a `Misc.isAsciiBased` function. That's only used for dumping debug data and works on Java charsets instead of our BitsCharset, but maybe it provides extra motivation that we might eventually want to have other `is*` members so we can ask questions about our supported charsets without having to make guesses based on the name.